pkg/crc/config/validations.go

```diff
@@ -41,6 +41,19 @@ func validateDiskSize(value interface{}) (bool, string) {
 	return true, ""
 }
 
+// validatePersistentDiskSize checks if provided disk size is valid in the config
+func validatePersistentDiskSize(value interface{}) (bool, string) {
```

Comment:
this can also be renamed to `validatePersistentVolumeSize` i think, since this is only used for the persistent volume config validation and not used for `disk-size` config option?

Reply:
yes this is also renamed.